src/BlockArea.cpp

```diff
@@ -441,7 +441,7 @@ bool cBlockArea::Read(cForEachChunkProvider * a_ForEachChunkProvider, int a_MinB
 		LOGWARNING("%s: MaxBlockY less than zero, adjusting to zero", __FUNCTION__);
 		a_MaxBlockY = 0;
 	}
-	else if (a_MaxBlockY > cChunkDef::Height)
+	else if (a_MaxBlockY >= cChunkDef::Height)
```

Comment:
I still think this is a wrong change, we need to be able to read areas with full chunk height (`a_MaxBlockY` is exclusive at this point due to line 425)